diff --git a/osdep/mac/meson.build b/osdep/mac/meson.build
index 2dc047176a..915d1d7c23 100644
--- a/osdep/mac/meson.build
+++ b/osdep/mac/meson.build
@@ -39,6 +39,7 @@ swift_compile = [swift_prog, swift_flags, '-module-name', 'swift',
                  '@INPUT@', '-I.', '-I' + source_root, '-I' + source_root / 'include',
                  '-I' + libplacebo.get_variable('includedir',
                             default_value: source_root / 'subprojects' / 'libplacebo' / 'src' / 'include'),
+                 '-I' + build_root / 'subprojects' / 'libplacebo' / 'src' / 'include',
                  '-I' + libavutil.get_variable('includedir')]
 
 swift_targets = custom_target('swift_targets',
